include/ibamr/FESurfaceDistanceEvaluator.h

```diff
@@ -254,6 +252,25 @@ class FESurfaceDistanceEvaluator
      */
     std::map<SAMRAI::pdat::CellIndex<NDIM>, std::set<libMesh::Elem*>, IBTK::CellIndexFortranOrder>
         d_cell_elem_neighbor_map;
+
+    /*!
+     * Map the node and the set of elements which comprise this node
+     */
+    std::map<libMesh::Node*, std::set<libMesh::Elem*> > d_node_to_elem;
+    /*!
+     * Map the edge and the set of elements which share this edge
+     */
+    std::map<std::pair<libMesh::Node*, libMesh::Node*>, std::set<libMesh::Elem*> > d_edge_to_elem;
+
+    /*!
+     * Map the element and its face normal
+     */
+    std::map<libMesh::Elem*, IBTK::VectorNd> d_elem_face_normal;
+
+    /*!
+     * In case of volume mesh, store the side_elements, triangle(edge) for 3D(2D), in a set
```

Comment:
`In case of volume mesh, store the side elements as TRI3(EDGE2) for three(two) dimensions.`